docs/source/modules.rst

```diff
@@ -113,6 +113,10 @@ Multi-Scale Intrinsic Distance (MSID)
 .. autoclass:: piq.MSID
     :members:
 
+Improved Precision and Recall (P&R)
+'''''''''''''''''''''''''''''''
```

Comment:
The formatting style for rst files requires the title and symbols below be of the same length.
```suggestion
Improved Precision and Recall (P&R)
'''''''''''''''''''''''''''''''''''
```